src/System.Text.Json/tests/Serialization/CustomConverterTests.BadConverters.cs

```diff
@@ -161,8 +161,8 @@ public static void ConverterReadTooLittle()
             }
             catch (JsonException ex)
             {
-                Assert.Contains("$.Level2.Level3s[0]", ex.ToString());
-                Assert.Equal(ex.Path, "$.Level2.Level3s[0]");
+                Assert.Contains("$.Level2.Level3s[1]", ex.ToString());
```

Comment:
What's causing this change?

Reply:
Converter validation changed because of the "OnReadEnumerable" to "ReadEnumable" above. Note that "bad converters" have the potential to cause random errors because of mismatched levels, tokens, etc so I added an additional check here for value converters (it can't read anything) and would like to add more including fixing the known issue with the serializer not being able to reason about CurrentDepth so we can verify depth is equal on the way out.